CHANGELOG.md

```diff
@@ -58,3 +58,4 @@ All notable, unreleased changes to this project will be documented in this file.
 - Fix checkout funds releasing task - #17198 by @IKarbowiak
 - Fixed 'healthcheck' middleware (`/health/` endpoint) not forwarding incoming traffic whenever the protocol wasn't HTTP (such as WebSocket or Lifespan) - #17248 by @NyanKiyoshi
 - Added support for the AWS_S3_URL_PROTOCOL environment variable - #17305 by @p-febis
+- Fix the issue of returning external shipping methods in a currency different from the checkout currency. - #17350 by @korycins
```

Comment:
But what was the issue? Let's ensure we have changelogs that explain before/after behavior if possible